### ISSUE TITLE
[Accessibility] Focus doesn't work in IE/Edge

### ISSUE THREAD

**Issue by zettca** [lumada-design/hv-uikit-react#1048]
**Describe the bug**
The current outline strategy we're using to highlight/Focus elements doesn't work in IE, because of `outline-style: auto`.

**To Reproduce**
1. Go to any page with focusable elements (like [List](https://pentaho.github.io/hv-uikit-react/?path=/story/components--list))
2. Tab through elements
3. Notice that the outline appears in every browser except IE/Edge

**Expected behavior**
Outline appearing properly

**Additional context**
This can be fixed using a conditional `isIE`, setting `outline-style` to `solid` instead of `auto`.
